package.json

```diff
@@ -64,7 +64,7 @@
     "lint:js": "eslint . --cache --max-warnings=0 --ext .js,.mjs",
     "lint:md": "remark . --quiet --frail",
     "lint:types": "tsc",
-    "prepare": "husky install && patch-package && npm run build",
```

Comment:
question

Don't we need `husky init` somewhere?
cf https://github.com/typicode/husky/releases/tag/v9.0.1